lib/main_common.pm

```diff
@@ -2177,9 +2177,9 @@ sub load_security_tests_crypt_core {
         loadtest "fips/openssl/openssl_fips_cipher";
         loadtest "fips/openssl/dirmngr_setup";
         loadtest "fips/openssl/dirmngr_daemon";    # dirmngr_daemon needs to be tested after dirmngr_setup
-        loadtest "fips/gnutls/gnutls_base_check";
-        loadtest "fips/gnutls/gnutls_server";
-        loadtest "fips/gnutls/gnutls_client";
+        loadtest "fips/gnutls/gnutls_base_check" if (!get_var("FIPS_ENV_MODE"));
+        loadtest "fips/gnutls/gnutls_server"     if (!get_var("FIPS_ENV_MODE"));
+        loadtest "fips/gnutls/gnutls_client"     if (!get_var("FIPS_ENV_MODE"));
```

Comment:
Why gnutls not supported in "FIPS_ENV_MODE" mode, is it by design?

Reply:
Good question, It makes sense I file a bug to double check then!

Reply:
As I know that the ENV mode is not pulled out/installed the fips library, it is just the **openssl** single module to export environment variable on FIPS mode, thus I think it's not in the **pure** FIPS mode, and I suppose the gnutls command will check the library which system is really working in FIPS mode. But it is also good to report a bug to track this.  :)

Reply:
Great,  Ben!
In that case, we can still carry out the tests, but don't need check the library .
Let me do some enhancement, and please help re-check again.